query/security/common/src/main/java/org/codice/ddf/admin/security/common/fields/wcpm/Realm.java

```diff
@@ -53,41 +52,53 @@ public Realm isRequired(boolean required) {
         return this;
     }
 
-    protected static final class LdapRealm extends StringField {
+    public static final class LdapRealm implements EnumValue<String> {
         public static final String LDAP = "ldap";
 
         public static final String DESCRIPTION =
                 "An LDAP used for authentication of users within it's database.";
 
-        public LdapRealm() {
-            super(LDAP, LDAP, DESCRIPTION);
+        @Override
+        public String enumTitle() {
+            return LDAP;
+        }
+
+        @Override
+        public String description() {
+            return DESCRIPTION;
         }
 
         @Override
-        public String getValue() {
+        public String value() {
             return LDAP;
         }
     }
 
-    protected static final class KarafRealm extends StringField {
+    public static final class KarafRealm implements EnumValue<String> {
         public static final String KARAF = "karaf";
 
         public static final String DESCRIPTION =
                 "The default realm. The karaf realm authenticates against the users.properties file.";
 
-        public KarafRealm() {
-            super(KARAF, KARAF, DESCRIPTION);
+        @Override
+        public String enumTitle() {
+            return KARAF;
```

Comment:
I just went through several enum implementations and it looks like the values returned from `enumTitle()` and `value()` are always the same (the static final string in the enum). Are there any cases where the title and value are different? If not, one of the methods should go away.